service_test.go

```diff
@@ -46,12 +46,24 @@ func TestBasicService(t *testing.T) {
 			}
 			return ctx
 		}).
-		Assess("should be reachable.", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
-			ma.Logger.Info("Starting ClusterIP")
-			reachability := matrix.NewReachability(pods, true)
-			testCase := matrix.TestCase{ToPort: 80, Protocol: v1.ProtocolTCP, Reachability: reachability, ServiceType: entities.ClusterIP}
-			wrong := matrix.ValidateOrFail(ma, model, &testCase, false)
-			if wrong > 0 {
+		Assess("should be reachable via cluster IP", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			ma.Logger.Info("Creating ClusterIP with ports in TCP and UDP.")
```

Comment:
is this log redundant?

Reply:
sorry just saw this. Yes and no.
Reason doing this way: as one clusterIP can have both tcp and udp port. Currently we are creating once, and testing tcp and udp in separate matrixes. 
Planning remove this line after combining two matrixes(TCP and UDP) into one.